index.php

```diff
@@ -143,8 +144,15 @@
 }
 
 if ($helper->is_username_set_in_auth_global_vars()) {
+    global $frm;
+    $loginFailCount = $helper->get_user_login_failCount($frm->username);
+
     echo $renderer->password_form($passwordform);
-} else {
+    if( isset($loginFailCount) &&  ($loginFailCount > 2) ) {
```

Comment:
Please fix the spacing on this line as per PSR-2 - https://www.php-fig.org/psr/psr-2/#5-control-structures